insights/specs/__init__.py

```diff
@@ -387,7 +387,7 @@ class Specs(SpecSet):
     modinfo_ixgbe = RegistryPoint()
     modinfo_veth = RegistryPoint()
     modinfo_vmxnet3 = RegistryPoint()
-    modinfo = RegistryPoint(multi_output=True)
+    modinfo = RegistryPoint(filterable=True)
```

Comment:
Since removing the `multi_output=True` will change the usage of the Parser, better to use another symbolink name.